packages/fxa-content-server/server/lib/routes/get-index.js

```diff
@@ -6,6 +6,7 @@
 
 const flowMetrics = require('../flow-metrics');
 const logger = require('../logging/log')('routes.index');
+const surveys = require('../../config/surveys.json');
```

Comment:
This would still require editing, pushing, and building in order update the list of survey correct?  Is FXA-2002 going to change this?

Reply:
This is what is described in FXA-2002, if we want something else we'll need to have a deeper discussion.

Reply:
> This is what is described in FXA-2002

Heh, that's true with a literal interpretation with FXA-2002's title.  But I think the original intention was to be able to change the configured surveys by updating the configurations and redeploying.  (At least that's what I gathered from reading the docs.)